flink-ml-parent/flink-ml-lib/src/test/java/org/apache/flink/ml/common/matrix/SparseVectorTest.java

```diff
@@ -0,0 +1,168 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.flink.ml.common.matrix;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * Test cases for SparseVector.
+ */
+public class SparseVectorTest {
+	private static final double TOL = 1.0e-6;
+	private SparseVector v1 = null;
+	private SparseVector v2 = null;
+
+	@Before
+	public void setUp() throws Exception {
```

Comment:
we can probably use static constructors like
```
  private static SparseVector v1 ...
  static {
    // ... construct SparseVectors
  }
```

Reply:
Since `setUp` is called before every tests, so I guess they are the same.

Reply:
if these vectors are not changed across tests. using `static final` might be a better idea. (based on the test they are not changed at all I am assuming? please correct me if I were making wrong assumption)

`@before` and `@after` invokes extra testing framework so it runs slower, it might be a good idea to lower overall test cost for Flink as a whole.

Reply:
thx, we have made the change.